STX.SPAL.Core.Tests.Unit/Services/Foundations/Assemblies/AssemblyServiceTests.Logic.GetApplicationPathsAssemblies.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+
+namespace STX.SPAL.Core.Tests.Unit.Services.Foundations.Assemblies
+{
+    public partial class AssemblyServiceTests
+    {
+        [Fact]
+        private async Task ShouldGetApplicationPathAssemblies()
```

Comment:
Do you await anything here?
If you do - then you need to `await` something inside of this method.
Otherwise just remove it. make it `void`

Reply:
Same thing for other methods.